lib/passwordLESS/passwordLESS.rb

```diff
@@ -0,0 +1,63 @@
+module PasswordLESS
+  require "passwordLESS/configuration"
+  require 'passwordLESS/gem/version'
+  class << self
+    attr_accessor :configuration
+    require 'net/http'
+    require 'uri'
+    require 'json'
+
+    def configuration
+      @configuration ||= Configuration.new
+    end
+
+    def configure
+      yield(configuration)
+    end
+
+    def send_otp(username)
+      request_body = { username: username }
+      initiate_request('/api/send_otp', request_body)
```

Comment:
@Deepan-kathir - Since `/api` would be common in all the request, can we move that with the base URL?